### DISCUSSION TITLE
some nextjs links not working in vercel deployed url

### DISCUSSION THREAD

**Discussion by ganeshkumars10d** [vercel/next.js#38058] [Help]
In local every route is working correctly and there is no issue in local but after deployed in vercel some links are not working completely ,and there is no errors in console its blank ,can someone say what is this issue

**Comment by icyJoseph** [comment]
Do you have a repository to take a look at? Code snippets? Perhaps a link to the deployment that is not working?

**Reply by ganeshkumars10d** [reply]
bro sorry for the late reply this issue is solved by upgrading the next version thank you
